### ISSUE TITLE
Can't resolve Error 1062

### ISSUE THREAD

**Issue by adamoyoung** [wandb/client#2112]
Details:
Ubuntu 16.04
wandb 0.10.27
python 3.6.13

I accidentally created two runs with the same run id (I know this is not normally possible). I now periodically receive the following error in my terminal:

`wandb: ERROR Error while calling W&B API: Error 1062: DUplicate entry '232268-lxly74ya' for key 'PRIMARY' (<Response [409]>)`

I have already tried the following:
- deleting my local wandb directory (with the local run directories)
- reinstalling wandb with pip
- deleting the run from the wandb web dashboard

However, I still keep getting the error! Any help would be appreciated.

**Comment by vanpelt** [comment]
Unfortunately, you have 2 options.

1. resume the existing run by adding `resume="allow"` to your wandb.init call
2. Use a different id by calling `wandb.util.generate_id()` and passing it into wandb.init

We have plans to free up deleted run ids, but currently we soft delete the run so the id remains in our database.

**Comment by adamoyoung** [comment]
So I have already deleted the run on the dashboard, and I'm not trying to re-run anything using the duplicate run id. Despite this, I keep getting the same error pop up in my terminal. Would I still be able to resume a run after I delete it?

**Comment by vanpelt** [comment]
@adamoyoung if you send me a link to the original run we can un-delete it for you if that's what you'd like.  I'm curious if you're able to use a different id in this case?  What's making you need to use the same id as the one you once deleted?

**Comment by adamoyoung** [comment]
Hm, I don't have a link to the original run because I deleted it from my dashboard (or at least, I thought I did). To be clear, I am not trying to use that id again: rather, every time I run something on my client, it gives me this error message over and over again, like in the image below. The id that is giving me trouble is "232268-lxly74ya"

![image](https://user-images.githubusercontent.com/22357382/116162188-4916da00-a6c3-11eb-970d-e864d2a2ac22.png)

**Comment by vanpelt** [comment]
Ahh, I see.  You just need to remove the file located at: `wandb/wandb-resume.json`

**Comment by adamoyoung** [comment]
Where is this file? I deleted my entire wandb/ directory in my repo for this project (~/msms) but I was still not able to stop the messages/

**Comment by vanpelt** [comment]
If you're not setting the id argument of wandb.init, then it's either reading a file named `wandb/wandb-resume.json` (relative to your script) or you have set the WANDB_RUN_ID environment variable.  Try typing `env | grey WANDB` to see if that's set in your environment.

If all else fails, you can run `wandb.init(id=wandb.util.generate_id())` to force the run to use a new id.

**Comment by adamoyoung** [comment]
OK, I tried running another script using `wand.init(id=wand.util.generate_id())`. This script worked (as it has previously). I also deleted the entire wandb directory in my folder. However, I am still periodically getting this error.

**Comment by adamoyoung** [comment]
Since the problem persists even after I delete the wandb directory and reinstall everything, I suspect that it is an issue on the server side, although I may be wrong.

**Comment by adamoyoung** [comment]
I tried deleting the project on the wandb dashboard, I think it solved the problem. It gave me the error message in the first screenshot below (presumably because I deleted the project, msms_pred, to which the bad run belonged). However, I noticed that after this error message was created, the previously-deleted msms_pred project was recreated on the dashboard, with the bad run being there (see second screenshot).
![image](https://user-images.githubusercontent.com/22357382/116281290-8fb61400-a757-11eb-9534-7abe66cf89f1.png)
![image](https://user-images.githubusercontent.com/22357382/116281424-b5431d80-a757-11eb-877a-93a475ff2b1a.png)

**Comment by vanpelt** [comment]
@adamoyoung I'm really confused as to what's happening.  Are you able to run new runs now?  If not it must be one of the following issues:

1. WANDB_RUN_ID is set in your environment
2. id=XXX is being set in your script
3. A file named `wandb/wandb-resume.json` exists relative to your script

If you're still seeing this issue, have you confirmed none of these things are true?

**Comment by adamoyoung** [comment]
I was always able to run new runs, it's just that I would get the error message that I posted (https://github.com/wandb/client/issues/2112#issuecomment-827199363) in my terminal, regardless of whether or not I was running anything. This message would appear periodically even when all my wandb runs were finished.

I checked all of the things that your mentioned. The error occurred when I was not even running anything, so I think 1. and 2. do not apply. I deleted the entire wandb directory and reinstalled wandb. I also searched my entire computer for wandb-resume.json and I could not find anything.

Just to reiterate: I am no longer seeing this issue. The only way I could solve it was by deleting the entire project (that contained the bad run_id) on the dashboard.
